<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1600</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1600</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>1600</width>
     <height>900</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1600</width>
     <height>900</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QGroupBox" name="gb_connection">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>481</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox{border:2px solid gray;border-radius:5px;margin-top: 0ex;color: ;background-color: transparent;} QGroupBox::title{subcontrol-origin: margin;subcontrol-position:top left;padding:0 2px;}
color: rgb(255, 255, 255);
</string>
    </property>
    <property name="title">
     <string>CONNECTION</string>
    </property>
    <widget class="QComboBox" name="cb_comports">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>STM32 COMPorts</string>
      </property>
     </item>
    </widget>
    <widget class="QComboBox" name="cb_baudrates">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>251</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <item>
      <property name="text">
       <string>STM32 Baudrates</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2400</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>34800</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>19200</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>57600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="btn_connect">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>30</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Connect</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_refresh">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>100</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_disconnect">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>30</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Disconnect</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="chb_status">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>170</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QCheckBox::indicator {
    border: 3px solid rgb(223, 0, 0);
	width: 25px;
	height: 25px;
	border-radius: 15px;
    background: rgb(223, 0, 0);
}
QCheckBox::indicator:hover {
    border: 3px solid rgb(223, 0, 0);
}
QCheckBox::indicator:checked {
    background: 3px solid rgb(60, 168, 31);
	border: 3px solid  rgb(60, 168, 31);	
	background-color: rgb(60, 168, 31);
}
</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_status">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>181</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_parameters">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>400</y>
      <width>481</width>
      <height>211</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{border:2px solid gray;
border-radius:5px;
margin-top: 0ex;
color: ;
background-color: transparent;}

QGroupBox::title
{subcontrol-origin:margin ;
subcontrol-position:top left;
padding:0 2px;}
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>PARAMETERS</string>
    </property>
    <widget class="QPushButton" name="btn_start">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_stop">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>20</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_page_the">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>101</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Page 1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_page_err">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Page 2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_time">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>170</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_time">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>170</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>t:</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="gb_response">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>620</y>
      <width>481</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QGroupBox
{border:2px solid gray;
border-radius:5px;
margin-top: 0ex;
color: ;
background-color: transparent;}

QGroupBox::title
{subcontrol-origin:margin ;
subcontrol-position:top left;
padding:0 2px;}
color: rgb(255, 255, 255);</string>
    </property>
    <property name="title">
     <string>RESPONSE</string>
    </property>
    <widget class="QLineEdit" name="le_the1">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_the1">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T1:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_the2">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T2:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_the2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_the3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T3:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_the3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_the4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>150</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T4:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_the4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>150</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_the5">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T5:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_the5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>190</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_the6">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>230</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>T6:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_the6">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>230</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_x">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>30</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Err1:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_x">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>30</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_z">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>110</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Err3:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_roll">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>150</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Err4:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_pitch">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Err5:</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_yaw">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>230</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Err6:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_pitch">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>190</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_y">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>70</y>
       <width>47</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Err2:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_roll">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>150</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_yaw">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>230</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_y">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>70</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="le_z">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>110</y>
       <width>113</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>15</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {

	border-radius: 5px;
	border: 2px solid rgb(27, 29, 35);
	padding-left: 10px;
}
QLineEdit:hover {
	border: 2px solid rgb(64, 71, 88);

}
QLineEdit:focus {
	border: 2px solid rgb(91, 101, 124);
}</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>120</y>
      <width>1081</width>
      <height>771</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page_the">
     <widget class="QWidget" name="formLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1061</width>
        <height>741</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formlayout_the"/>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Trang 1</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_err">
     <widget class="QWidget" name="formLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>1061</width>
        <height>741</height>
       </rect>
      </property>
      <layout class="QFormLayout" name="formlayout_err"/>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>10</y>
        <width>47</width>
        <height>13</height>
       </rect>
      </property>
      <property name="text">
       <string>Trang 2</string>
      </property>
     </widget>
    </widget>
   </widget>
   <zorder>stackedWidget</zorder>
   <zorder>gb_connection</zorder>
   <zorder>gb_parameters</zorder>
   <zorder>gb_response</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
